### ISSUE TITLE
Results on ontonotes

### ISSUE THREAD

**Issue by LearningPytorch** [sheng-z/figet#1]
Hi I'm trying to replicate your numbers on Ontonotes but running for 15 Epochs which is default, gives me a performance of `52.98, 69.58, 62.95` which is much lower than the numbers in the paper. Can you tell me if I'm doing something wrong? Thanks very much

**Comment by sheng-z** [comment]
Have you tried Step 4? I have no problem reproducing the similar results from the scratch.
```
06/17/2018 06:26:44 | Tuned | dev acc. (54.18, 68.89, 63.25) | test acc. (53.35, 72.62, 66.10) |
```
Please note the released datasets don't have document-level contexts because of the copy right issue. The results you are able to reproduce are those w/o document-level contexts.

**Comment by LearningPytorch** [comment]
>Have you tried Step 4? I have no problem reproducing the similar results from the scratch.

For how many epochs did you train? 15? I'm not able to reach that. I tried 15, 40 and 200. I've Pytorch 0.20 and Cuda 80.

>Please note the released datasets don't have document-level contexts because of the copy right issue. The results you are able to reproduce are those w/o document-level contexts.

Is there a way to obtain these for research purposes?

>06/17/2018 06:26:44 | Tuned | dev acc. (54.18, 68.89, 63.25) | test acc. (53.35, 72.62, 66.10) |

Is this with document-level context?

**Comment by LearningPytorch** [comment]
These're my results on Ontonotes with 15 Epochs and after tuned:
`06/18/2018 10:43:46 | Tuned | dev acc. (53.18, 67.67, 61.46) | test acc. (52.90, 69.40, 63.18) |
`
And on Wiki after 15 Epochs and tuned:
`06/18/2018 02:55:57 | Tuned | dev acc. (57.91, 82.28, 81.77) | test acc. (58.26, 76.73, 73.54) |`

**Comment by LearningPytorch** [comment]
@sheng-z not sure if you saw my responses? I'm trying to replicate your numbers and your help is appreciated. I'm doing some experiments myself and will cite your work if I'm successful in doing these experiments. Thanks.

**Comment by sheng-z** [comment]
Hi, the result I sent to you earlier is from the default setting without document-level context. Not sure why you can't replicate similar results. The only suggestion I can have is to try a different random seed.
Sorry if my code is not helpful to you.
